src/app/changeUsernameBanner/changeUsernameBanner.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="container tos-banner text-center" data-cy="changeUsernameBanner">
+  <div class="container-fluid" fxLayout="" fxLayoutAlign="center center">
```

Comment:
avoid container and container-fluid (unless other banners also use this), it's from legacy bootstrap. `fxLayout=""` probably isn't needed

Reply:
they do, but looked the same once I removed them